
# dati codificati
uint8[22] teapotData

